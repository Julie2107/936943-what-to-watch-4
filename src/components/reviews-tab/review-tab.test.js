import React from "react";
import renderer from "react-test-renderer";

import ReviewsTab from "./reviews-tab.jsx";

const comments = [
  {author: `Amanda Greever`,
    date: `November 18, 2015`,
    message: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
    rating: 2},
  {author: `Amanda Greever`,
    date: `November 18, 2015`,
    message: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
    rating: 2},
];

it(`Render reviews tab`, () => {
  const tree = renderer
    .create(<ReviewsTab
      reviews = {comments}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
