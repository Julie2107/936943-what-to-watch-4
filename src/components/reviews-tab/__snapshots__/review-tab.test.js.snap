// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render reviews tab 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Amanda Greever
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            November 18, 2015
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Amanda Greever
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            November 18, 2015
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2
      </div>
    </div>
  </div>
</div>
`;
